src/test/resources/application.properties

```diff
@@ -8,12 +8,13 @@ serviceConfig.storage=local
 serviceConfig.accessKey=test
 serviceConfig.secretKey=test
 serviceConfig.bucket=test
+serviceConfig.baseurl=test
 
 # minting option
 serviceConfig.minter=pid
 
 # Spring profile to use. Set to 'stage' when testing integration tests for S3.
-spring.profiles.active=dev
+spring.profiles.active=stage
```

Comment:
@ehanson8 This should also be set back to dev. Thanks!

Reply:
@osmandin Fixed